support-modules/acquisition-events/src/main/scala/com/gu/support/acquisitions/utils/retry.scala

```diff
@@ -0,0 +1,17 @@
+package com.gu.support.acquisitions.utils
+
+import scala.concurrent.{Future, ExecutionContext}
+import cats.data.EitherT
+import cats.instances.future._
+
+object Retry {
+  def apply[T](max: Int)(f: => EitherT[Future, String, Unit])(implicit ec: ExecutionContext): EitherT[Future, String, Unit] = {
```

Comment:
unused type parameter here, was it intended for the result type?
It's not important as the two uses of this function have the same type, but you could make the Left and Right types generic

Reply:
oh yeah! must have missed that - I'll make the `Left` and `Right` types generic as that feels quite nice!